decidim-admin/app/views/decidim/admin/user_groups/index.html.erb

```diff
@@ -24,7 +25,13 @@
         <tbody>
           <% @user_groups.each do |user_group| %>
             <tr data-user-group-id="<%= user_group.id %>">
-              <td><%= user_group.name %></td>
+              <% if user_group.nickname.present? %>
+                <td><%= link_to user_group.name, decidim.profile_path(user_group.nickname) %></td>
```

Comment:
I guess, you could avoid all this If else statement, if you would use a [`link_to_if`](https://apidock.com/rails/ActionView/Helpers/UrlHelper/link_to_if) statement. 
```
<td><%= link_to_if user_group.nickname.present?, user_group.name, decidim.profile_path(user_group.nickname) %></td>
<td><%= link_to_if user_group.nickname.present?, user_group.nickname, decidim.profile_path(user_group.nickname) %></td>
```